app/vminsert/relabel/relabel.go

```diff
@@ -16,11 +18,14 @@ import (
 var (
 	relabelConfig = flag.String("relabelConfig", "", "Optional path to a file with relabeling rules, which are applied to all the ingested metrics. "+
 		"The path can point either to local file or to http url. "+
-		"See https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#relabeling for details. The config is reloaded on SIGHUP signal")
+		"See https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#relabeling for details. The config is reloaded on SIGHUP signal or"+
+		"with periodicity that is defined via -relabel.configCheckInterval")
 
 	usePromCompatibleNaming = flag.Bool("usePromCompatibleNaming", false, "Whether to replace characters unsupported by Prometheus with underscores "+
 		"in the ingested metric names and label names. For example, foo.bar{a.b='c'} is transformed into foo_bar{a_b='c'} during data ingestion if this flag is set. "+
 		"See https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels")
+	relabelConfigCheckInterval = flag.Duration("relabelConfigCheckInterval", 0, "Interval for checking for changes in '-relabelConfig' file. "+
```

Comment:
the flag name should be `relabel.configCheckInterval`, like on L:22 above.

Reply:
Ok, then `"with periodicity that is defined via -relabel.configCheckInterval"` has to be changed accordingly. 
Please verify that all readmes changes are also correct.

Reply:
fyi, the command description has been adjusted in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/7c0c8cc702fc7c14e6fc1e982fac60af783f3a4f#diff-960f291ff3acc25285cffb51be1476a30dc4239d7e34ef8a94ba0f3efef7dd16R22. Also, in that commit all flags were synced with the actual binaries.